src/gluonts/evaluation/backtest.py

```diff
@@ -36,11 +35,10 @@
 from gluonts.model.predictor import GluonPredictor, Predictor
 from gluonts.support.util import maybe_len
 from gluonts.transform import TransformedDataset
-from mxnet.ndarray import NDArray
 
 
 def make_evaluation_predictions(
-    dataset: Dataset, predictor: Predictor, num_samples: int
+    dataset: Dataset, predictor: Predictor, num_samples: int,
```

Comment:
I think these changes shouldn't be commited.

Reply:
sure, since I moved all changes to other files than `backtest.py` I will simply revert all changes made to backtest to when I started the implementation instead.